app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    sepp_command = command.split()
+
+    source = sepp_command[1]
```

Comment:
Consider using unpacking to assign values from `sepp_command` to `source` and `destination`. This is a more efficient and cleaner approach.